CMakeLists.txt

```diff
@@ -97,7 +131,14 @@ if ("CUDA" STREQUAL "${THRUST_DEVICE_SYSTEM}")
   set(THRUST_HIGHEST_COMPUTE_ARCH 75)
   set(THRUST_KNOWN_COMPUTE_ARCHS 30 32 35 50 52 53 60 61 62 70 72 75)
 
-  option(THRUST_DISABLE_ARCH_BY_DEFAULT "If ON, then all CUDA architectures are disabled on the initial CMake run." OFF)
+  if ("Feta" STREQUAL "${CMAKE_CUDA_COMPILER_ID}")
```

Comment:
If Feta cannot support multiple archs at once, we can use different logic to specifying the target arch under feta:

```cmake
set(THRUST_COMPUTE_ARCH "30" CACHE STRING "Target compute architecture")
set_property(CACHE THRUST_COMPUTE_ARCH PROPERTY STRINGS ${THRUST_KNOWN_COMPUTE_ARCHS})
```

That will add a CMake option `THRUST_COMPUTE_ARCH` that will allow selection from the values in `THRUST_KNOWN_COMPUTE_ARCHS` when using ccmake/cmake-gui, while limiting the selection to a single value.

Reply:
Okay sounds fine.

Reply:
How would this work with NVCC? You use this to just target one compute arch? I'm not sure I like the idea of having different styles for different compilers.

Reply:
Probably best to leave this as is then.